app/services/alerts/framework_adapter.rb

```diff
@@ -19,6 +19,11 @@ def content
       adapter_instance.content
     end
 
+    def benchmark_dates
+      adapter_class(@alert_type).new(alert_type: @alert_type, school: @school, analysis_date: @analysis_date, aggregate_school: @aggregate_school)
```

Comment:
Is this line needed? We're using adapter instance below.

Reply:
No, should be nobbled